src/main/java/uk/nhs/digital/ps/migrator/model/hippo/XlsxReader.java

```diff
@@ -91,18 +94,35 @@ public Integer getColumnIndex(String columnName){
 
         return this.headerNameIndexPairs.get(cleanHeader(columnName));    
     }
-
+  
     /**
      * Returns string cell value. Apache code will throw exception if the cell is not of type string.
      */        
     public String getCellValue(String columnName, Row row){
-        return row.getCell(getColumnIndex(columnName)).getStringCellValue().trim().replace("\n", "\\n").replace("\r", "\\r");
+        Cell cell = row.getCell(getColumnIndex(columnName));
+        if (cell == null)
+            return null;
+
+        String rawValue = cell.getStringCellValue();
+        if (rawValue == null)
+            return null;
+
+
+        return TextHelper.escapeSpecialCharsForJson(rawValue);
     }
 
     /**l
```

Comment:
I think this redundant 'l' is a result of locking the machine via WinKey+L - happens to me all the time :smile: (I know it's from another commit, just spotted it now).

Reply:
From now on I shall call you Hawkeye.